modules/installation-about-mirror-registry.adoc

```diff
@@ -30,7 +30,7 @@ endif::[]
 
 [IMPORTANT]
 ====
-The internal registry of the {product-title} cluster cannot be used as the target registry because it does not support pushing without a tag, which is required during the mirroring process.
+The {product-registry} of the {product-title} cluster cannot be used as the target registry because it does not support pushing without a tag, which is required during the mirroring process.
```

Comment:
This might be overkill to have both `{product-registry}` and `{product-title}` ("OpenShift" is already in the `{product-registry}` attribute). Would it still make sense instead to say, "The {product-registry} cannot be used..."?